packages/livekit-rtc/src/participant.ts

```diff
@@ -95,6 +100,10 @@ export abstract class Participant {
   get kind(): ParticipantKind | undefined {
     return this.info.kind;
   }
+
+  get disconnectReason(): DisconnectReason | undefined {
+    return this.info.disconnectReason;
```

Comment:
info.disconnectReason might be set to unknown (since not nullable)

we'd want to check and return undefined if so